assets/src/styles/blocks/core-overrides/BlockSpacing.scss

```diff
@@ -23,6 +22,20 @@
   }
 }
 
+.page-content > .wp-block-media-text {
+  margin-top: $sp-1;
+  margin-bottom: $sp-2;
+
+  @include large-and-up {
+    margin-top: $sp-2;
+    margin-bottom: $sp-4;
+  }
+
+  &:first-of-type {
```

Comment:
Shouldn't it be `first-child` instead of `first-of-type`?

Reply:
Yep! Makes more sense !